includes/class-amp-travel-cpt.php

```diff
@@ -198,6 +225,42 @@ public function adventure_meta_box_html() {
 
 		<label for='amp_travel_price'><?php esc_html_e( 'Price (USD)', 'travel' ); ?></label>
 		<input id='amp_travel_price' name='amp_travel_price' value='<?php echo esc_attr( $price ); ?>'>
+		<label for='amp_travel_images'><?php esc_html_e( 'Additional carousel images.', 'travel' ); ?></label>
+		<div class="editor-post-featured-image" id="amp-travel-images-wrap">
+			<?php
+			foreach ( $images as $image ) {
+				if ( wp_attachment_is_image( $image ) ) {
+					$this->image_template( $image );
+				}
+			}
+			?>
+		</div>
+		<button type="button" id="amp_travel_images" class="travel-image-select button" data-title="<?php esc_attr_e( 'Select carousel images.', 'travel' ); ?>" data-text="<?php esc_attr_e( 'Use selection', 'travel' ); ?>"><?php esc_html_e( 'Select Images', 'travel' ); ?></button>
+		<script type="text/html" id="travel-images-tmpl">
+			<?php $this->image_template(); ?>
+		</script>
+		<?php
+	}
+
+	/**
+	 * Output a a template for a single image item.
+	 *
+	 * @param int $id The id of the image.
+	 */
+	public function image_template( $id = null ) {
+		if ( null === $id ) {
+			$image_src = '{{url}}';
+			$id        = '{{id}}';
+		} else {
+			$image_src = wp_get_attachment_image_url( $id, 'medium' );
+		}
+
+		?>
+		<div class="travel-images-preview" data-id="{{id}}" id="travel-image-<?php echo esc_attr( $id ); ?>">
+			<img src="<?php echo esc_attr( $image_src ); ?>" class="travel-image-control-preview">
```

Comment:
@kienstra It should, however since I'm using it as a template as well, using `esc_url` breaks `{{url}}` in template form.
What I should do is `esc_url` if `ID` is set. hmm. I'll make a change.